action.yml

```diff
@@ -24,6 +24,10 @@ inputs:
         - base: backport PR's base branch
         - number: original PR's number
     default: "backport-<%= number %>-to-<%= base %>"
+  team_reviews:
+    description: Whether to request reviews from the team.
+    required: false
+    default: "release"
```

Comment:
how does the `default` option work here, will this still automatically tag the release team?

Reply:
if not overridden it will still tag the release team, but I plan to override it for jetbrains and cody so that it wont

Reply:
great ok!